flax/linen/sequential.py

```diff
@@ -0,0 +1,40 @@
+# Copyright 2021 The Flax Authors.
```

Comment:
Can we call this file combinators.py instead in case we add other things like this.